lib/check-tags.js

```diff
@@ -0,0 +1,41 @@
+var log = require('./out')({});
+
+function checkTags(app, mod, name) {
+  if (mod.tags && mod.tags !== '' && typeof mod.tags === 'string') {
+    mod.tags = mod.tags.split(',');
+  }
+
+  function checkMatch (A, B) {
+    return A.filter(function(tag) {
+      return (B.indexOf(tag) !== -1);
```

Comment:
Is there a reason not to use an arrow function here? This seems like the perfect place for one (see earlier comment).

Also why have `checkMatches` calling `checkMatch`? This code checks for all matches, not just one. I don't see the need for two functions.